xorgauth/templates/base.html

```diff
@@ -40,6 +40,24 @@
             {% else %}
             <li class="nav-item"><a class="nav-link" href="{% url 'login' %}">{% trans 'Log in' %}</a></li>
             {% endif %}
+            <form class="navbar-form navbar-left" action="{% url 'set_language' %}" method="post">{% csrf_token %}
+                <div class="form-group">
+                    <input name="next" type="hidden" value="{{ redirect_to }}" />
```

Comment:
While reviewing this, I wondered what this input variable does (when I test this PR, "redirect_to" seems to be always empty). Nevertheless this code directly comes from Django's documentation (https://docs.djangoproject.com/en/2.0/topics/i18n/translation/ ) so it may be useful in an case.
This looks good to me. Thanks !